tests/2node.yml

```diff
@@ -14,6 +14,10 @@ items:
       local_ip: 12.12.12.1/24
       peer_intf: eth1
       peer_ip: 12.12.12.2/24
+    - uid: 2
```

Comment:
I think `uid:2` is already used, try making it 3 to avoid confusion.
As for the testing, try giving both ends of the link an IP address. I'm assuming `ping -I <sourceIntf> <destIP>` should force the traffic down the veth pair. Might be worth checking with tcpdump.

Reply:
fixed the uid, though the veth is created fine, having same subnet on two interfaces would not work properly without moving the interfaces into separate netns.

143: eth3@eth2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP 
    link/ether ba:7e:81:44:27:86 brd ff:ff:ff:ff:ff:ff
    inet 13.13.13.2/24 brd 13.13.13.255 scope global eth3
       valid_lft forever preferred_lft forever
144: eth2@eth3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP 
    link/ether 52:fd:54:06:4e:50 brd ff:ff:ff:ff:ff:ff
    inet 13.13.13.1/24 brd 13.13.13.255 scope global eth2
       valid_lft forever preferred_lft forever